config/config.yaml

```diff
@@ -43,23 +50,25 @@ diffexp:
   exclude:
   # model for sleuth differential expression analysis
   models:
-    model_X:
-      full: ~condition + batch_effect
-      reduced: ~batch_effect
+    model_Condition:
+      full: ~condition
+      #full: ~condition + batch_effect
+      reduced: '~1'
+      #reduced: '~1'# in case if it takes as numerical
       # Binary valued covariate that shall be used for fold change/effect size
       # based downstream analyses.
       primary_variable: condition
       # base level of the primary variable (will be considered as denominator 
       # in the fold change/effect size estimation).
-      base_level: untreated
+      base_level: Non_resp
```

Comment:
Let's not change the configuration example.

Reply:
Noted

Reply:
Still different from before.

Reply:
Noted. Changed